project.clj

```diff
@@ -33,14 +33,18 @@
   :source-paths ["src/clj"]
   :resource-paths ["resources"]
 
-  :cljsbuild {
-    :builds [{:id "cloujera"
-              :source-paths ["src/cljs"]
-              :compiler {
-                :output-to "resources/public/js/cloujera.js"
-                :output-dir "resources/public/js/out"
-                :optimizations :none
-                :source-map true}}]}
+  :cljsbuild {:builds [{:id "dev"
+                        :source-paths ["src/cljs" "dev-config"]
```

Comment:
the config for the `cljs` backend URI now lives in files in `./dev-config` and `./prod-config`